### PR TITLE
Fix logo and update dependencies

### PR DESCRIPTION
**Pull request by obynio**
Hello guys,

I just noticed that the status page has a link to one of our network diagnose tool. The Algolia logo was broken so here is a PR to fix it. I also updated the dependencies to fix the vulns in the deprecated packages.

Changelog:

* Embed the Algolia logo SVG in the sources to ensure the url does not break again.
* Update dependencies
* Fix the `lodash` module path in the sources
* Remove `json-loader` since webpack can now handle JSON natively 
https://github.com/webpack-contrib/file-loader/issues/259

Sorry to assign you this PR but I don't know who to ask since Vincent left. I did what I can, but I'm not proficient in JS so feel free to comment :)

### CODE REVIEW

**Comment by obynio** [comment]
Just failing linting, but the rest is working 👍

File: src/diagnostics/empty-search.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ function emptySearch(cb) {
   var start = Date.now();
   index.search(function searchDone(err, content) {
     var humanize = require('humanize-number');
-    var pluck = require('lodash/collection/pluck');
+    var pluck = require('lodash/fp/pluck');
```
Comment (Haroenv) [inline src/diagnostics/empty-search.js]: I'm not sure /collection/ is replaced by fp. I thought those have different arguments. Did you try this code?

**Comment by Haroenv** [comment]
I get quite different results with your PR:

```
==============
ERROR: User-agent
==============

  - error.message > undefined
  - error.stack > no stack information


==============
request ip
==============

  - request ip > err: cannot get ip, err was: Error: Request has been terminated
Possible causes: the network is offline, Origin is not allowed by Access-Control-Allow-Origin, the page is being unloaded, etc.


==============
proxy detection
==============

  - proxy > err: cannot detect proxy, err was: Error: Request has been terminated
Possible causes: the network is offline, Origin is not allowed by Access-Control-Allow-Origin, the page is being unloaded, etc.


==============
Geolocation (browser + //nominatim.openstreetmap.org/reverse)
==============

  - display_name > err: could not get position, error was Origin does not have permission to use Geolocation service


==============
ERROR: Navigation timing from current page (ms)
==============

  - error.message > undefined
  - error.stack > no stack information


==============
`/favicon.ico` download
==============

# 0
  - website > www.google.com
  - time to download (ms) > 60

# 1
  - website > www.baidu.com
  - time to download (ms) > 736

# 2
  - website > www.yandex.ru
  - time to download (ms) > 200

# 3
  - website > www.google.com.br
  - time to download (ms) > 50

# 4
  - website > www.jumia.com
  - time to download (ms) > 268

# 5
  - website > www.google.co.jp
  - time to download (ms) > 50


==============
ERROR: Resource timing
==============

  - error.message > undefined
  - error.stack > no stack information


==============
Doing an empty search on latency:contacts (6be0576ff61c053d5f9a3225e2a90f76)
using JavaScript API client `algoliasearch@3.35.1`
==============

  - object ids > false,false,false,false,false,false,false,false
  - nb hits > 1
  - time (ms) > 133
  - full log > 
algoliasearch init done, {"applicationID":"latency","apiKey":"6be0576ff61c053d5f9a3225e2a90f76","hosts":{"read":["https://latency-dsn.algolia.net","https://latency-3.algolianet.com","https://latency-1.algolianet.com","https://latency-2.algolianet.com"],"write":["https://latency.algolia.net","https://latency-3.algolianet.com","https://latency-1.algolianet.com","https://latency-2.algolianet.com"]},"_timeouts":{"connect":1000,"read":2000,"write":30000},"_hostIndexes":{"read":0,"write":0},"_timeoutMultiplier":1,"_shuffleResult":[3,1,2],"extraHeaders":{},"cache":{},"_ua":"Algolia for JavaScript (3.35.1); Browser","_useCache":true,"_useFallback":true} +0ms
algoliasearch:/1/indexes/contacts/query request start +1ms
algoliasearch:/1/indexes/contacts/query method: POST, url: https://latency-dsn.algolia.net/1/indexes/contacts/query, headers: {"x-algolia-agent":"Algolia for JavaScript (3.35.1); Browser","x-algolia-application-id":"latency","x-algolia-api-key":"6be0576ff61c053d5f9a3225e2a90f76"}, timeouts: NaN +1ms
algoliasearch:/1/indexes/contacts/query error: Network issue, see err.more for details, stack: o@https://cdn.jsdelivr.net/npm/algoliasearch@3/dist/algoliasearch.min.js:4:3584
r@https://cdn.jsdelivr.net/npm/algoliasearch@3/dist/algoliasearch.min.js:4:3850
i@https://cdn.jsdelivr.net/npm/algoliasearch@3/dist/algoliasearch.min.js:3:30376 +30ms
algoliasearch:/1/indexes/contacts/query retrying request +0ms
algoliasearch:/1/indexes/contacts/query method: POST, url: https://latency-3.algolianet.com/1/indexes/contacts/query, headers: {"x-algolia-agent":"Algolia for JavaScript (3.35.1); Browser","x-algolia-application-id":"latency","x-algolia-api-key":"6be0576ff61c053d5f9a3225e2a90f76"}, timeouts: NaN +0ms
algoliasearch:/1/indexes/contacts/query received response: statusCode: 200, computed statusCode: 200, headers: "cache-control: no-store, no-cache\r\ncontent-length: 469\r\ncontent-type: application/json; charset=UTF-8\r\n" +95ms



==============
Network measurements using lognormal/boomerang
==============

  - page load time (ms) > 1,980
  - ttfb (ms) > 5
  - bandwidth > 151.1 MB/s
  - latency (ms) > 4
  - ipv6 google.com/images/logo.png time (ms) > NA


==============
END
==============
```

vs live version:

```
==============
User-agent
==============

  - browser-name > Safari
  - browser-major > 13
  - browser-version > 13.2
  - os-name > Mac OS
  - os-version > 10.15.2
  - timestamp > 1582708742328
  - timezone > Europe/Paris
  - timezone offset from UTC (minutes) > -60
  - local time > Wed Feb 26 2020 10:19:02 GMT+0100 (CET)
  - device-type > desktop


==============
request ip
==============

  - request ip > 84.14.205.83


==============
proxy detection
==============

  - proxy > not found


==============
Geolocation (browser + //nominatim.openstreetmap.org/reverse)
==============

  - display_name > 55, Rue d'Amsterdam, Quartier Saint-Georges, 9th Arrondissement, Paris, Ile-de-France, Metropolitan France, 75009, France
  - latitude > 48.88041935086184
  - longitude > 2.3270472865189022


==============
Navigation timing from current page (ms)
==============

  - connect > 0
  - ssl connect > n/a
  - dns > 0
  - ttfb > 240
  - response download > 12
  - total > 252


==============
`/favicon.ico` download
==============

# 0
  - website > www.google.com
  - time to download (ms) > 55

# 1
  - website > www.baidu.com
  - time to download (ms) > 1,026

# 2
  - website > www.yandex.ru
  - time to download (ms) > 313

# 3
  - website > www.google.com.br
  - time to download (ms) > 75

# 4
  - website > www.jumia.com
  - time to download (ms) > 198

# 5
  - website > www.google.co.jp
  - time to download (ms) > 56


==============
Timing https://latency-dsn.algolia.net/diag (ms)
==============

# 0
  - connect > 0
  - ssl connect > n/a
  - dns > 0
  - ttfb > 7
  - response download > 1
  - total > 8

# 1
  - connect > 0
  - ssl connect > n/a
  - dns > 0
  - ttfb > 10
  - response download > 0
  - total > 10

# 2
  - connect > 0
  - ssl connect > n/a
  - dns > 0
  - ttfb > 9
  - response download > 1
  - total > 10


==============
Timing https://latency-1.algolianet.com/diag (ms)
==============

# 0
  - connect > 61
  - ssl connect > 47
  - dns > 22
  - ttfb > 13
  - response download > 1
  - total > 101

# 1
  - connect > 0
  - ssl connect > n/a
  - dns > 0
  - ttfb > 14
  - response download > 1
  - total > 15

# 2
  - connect > 0
  - ssl connect > n/a
  - dns > 0
  - ttfb > 13
  - response download > 0
  - total > 13


==============
Timing https://latency-2.algolianet.com/diag (ms)
==============

# 0
  - connect > 36
  - ssl connect > 28
  - dns > 19
  - ttfb > 6
  - response download > 1
  - total > 63

# 1
  - connect > 0
  - ssl connect > n/a
  - dns > 0
  - ttfb > 8
  - response download > 0
  - total > 8

# 2
  - connect > 0
  - ssl connect > n/a
  - dns > 0
  - ttfb > 8
  - response download > 0
  - total > 8


==============
Timing https://latency-3.algolianet.com/diag (ms)
==============

# 0
  - connect > 61
  - ssl connect > 47
  - dns > 2
  - ttfb > 13
  - response download > 1
  - total > 81

# 1
  - connect > 0
  - ssl connect > n/a
  - dns > 0
  - ttfb > 16
  - response download > 0
  - total > 16

# 2
  - connect > 0
  - ssl connect > n/a
  - dns > 0
  - ttfb > 15
  - response download > 0
  - total > 15


==============
Doing an empty search on latency:contacts (6be0576ff61c053d5f9a3225e2a90f76)
using JavaScript API client `algoliasearch@3.35.1`
==============

  - object ids > 400091041
  - nb hits > 1
  - time (ms) > 21
  - full log > 
algoliasearch init done, {"applicationID":"latency","apiKey":"6be0576ff61c053d5f9a3225e2a90f76","hosts":{"read":["https://latency-dsn.algolia.net","https://latency-2.algolianet.com","https://latency-3.algolianet.com","https://latency-1.algolianet.com"],"write":["https://latency.algolia.net","https://latency-2.algolianet.com","https://latency-3.algolianet.com","https://latency-1.algolianet.com"]},"_timeouts":{"connect":1000,"read":2000,"write":30000},"_hostIndexes":{"read":0,"write":0},"_timeoutMultiplier":1,"_shuffleResult":[2,3,1],"extraHeaders":{},"cache":{},"_ua":"Algolia for JavaScript (3.35.1); Browser","_useCache":true,"_useFallback":true} +0ms
algoliasearch:/1/indexes/contacts/query request start +2ms
algoliasearch:/1/indexes/contacts/query method: POST, url: https://latency-dsn.algolia.net/1/indexes/contacts/query, headers: {"x-algolia-agent":"Algolia for JavaScript (3.35.1); Browser","x-algolia-application-id":"latency","x-algolia-api-key":"6be0576ff61c053d5f9a3225e2a90f76"}, timeouts: NaN +0ms
algoliasearch:/1/indexes/contacts/query received response: statusCode: 200, computed statusCode: 200, headers: "cache-control: no-store, no-cache\r\ncontent-length: 469\r\ncontent-type: application/json; charset=UTF-8\r\n" +16ms



==============
Network measurements using lognormal/boomerang
==============

  - page load time (ms) > 5,054
  - ttfb (ms) > 240
  - bandwidth > 5.5 MB/s
  - latency (ms) > 205
  - ipv6 google.com/images/logo.png time (ms) > n/a


==============
END
==============


```

**Comment by Haroenv** [comment]
managed to get most passing now, still a few different

**Comment by francoischalifour** [comment]
I didn't know about this project haha

I made a few changes:

- Use logo for dark background
- Use `.eslintignore` to ignore linting on build files
- Use pined deps as per Renovate recommendation for apps (useful because this is an unmaintained project)

**Pull request merged**